iOS/PixelDefinitions/pixels/widget_reporting.json5

```diff
@@ -0,0 +1,42 @@
+// See https://app.asana.com/1/137249556945/project/392891325557410/task/1210882421460693
+{
+    'm_widget-report': {
+        description: 'Fires daily to report configured widgets on the device, including widget kinds and families. This pixel can be enabled and disabled using the iOSBrowserConfig.widgetReporting feature flag which is off by default.',
+        owners: ['brindy'],
+        triggers: ['startup'],
+        suffixes: ['form_factor'],
+        parameters: [
+            'appVersion',
+            {
+                key: 'enabled_widgets',
+                description: 'A comma separated list of enabled widgets in the format WidgetName-Family. The widget name is defined statically in code so will not contain user inforation.  The family is defined by iOS',
+                type: 'string',
+                pattern: '([A-Za-z]+-(systemSmall|systemMedium|systemLarge))(,[A-Za-z]+-(systemSmall|systemMedium|systemLarge))*',
+                examples: [
+                    'QuickActionWidget-systemSmall',
+                    'QuickActionWidget-systemSmall,FavoritesWidget-systemLarge',
+                    'VPNStatusWidget-systemSmall,QuickActionsWidget-systemSmall,FavoritesWidget-systemMedium,FavoritesWidget-systemLarge',
+                ],
+            },
+        ],
+    },
+    'm_widget-report-failure': {
+        description: 'Fires when widget configuration reporting fails due to an error retrieving current widget configurations. This pixel can be enabled and disabled using the iOSBrowserConfig.widgetReporting feature flag which is off by default.',
+        owners: ['brindy'],
+        triggers: ['startup'],
+        suffixes: ['form_factor'],
+        parameters: [
+            'appVersion',
+            {
+                key: 'e',
+                description: 'The error code if present',
+                type: 'string',
+            },
+            {
+                key: 'd',
+                description: 'The domain of the error',
+                type: 'string',
+            },
```

Comment:
actually I think these are not the only parameters that can be sent, we recursively check for error's underlying error and send them too;
on one of my branches ([jacek/startup-pixels](https://github.com/duckduckgo/apple-browsers/blob/jacek/startup-pixels/iOS/PixelDefinitions/params_dictionary.json5)) I actually added these parameters so you can use them by "key". Check it out, note though I haven't validated it yet.